One Pace/Season 18/One Pace - S18E17 - Sogeking.nfo

```diff
@@ -1,25 +1,29 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<!--created on 2023-09-13 13:00:51 - tinyMediaManager 4.3.13-->
+<!--created on 2024-02-03 02:46:33 - tinyMediaManager 4.3.14-->
 <episodedetails>
-  <title>Showdown Between Cooks! Sanji vs Ramen Kenpo</title>
+  <title>Sogeking</title>
   <originaltitle/>
   <showtitle>One Pace</showtitle>
   <season>18</season>
-  <episode>259</episode>
+  <episode>17</episode>
   <displayseason>-1</displayseason>
   <displayepisode>-1</displayepisode>
   <id/>
   <ratings/>
   <userrating>0.0</userrating>
-  <plot>Kalifa updates the CP9 on the situation with Franky, Sanji, and Usopp. Sanji tells Franky and Usopp to go on without him so he can fight Wanze and his "Ramen Kenpo". Franky begins to fight Nero, CP9's "Weasel", and Sogeking meets with Robin.</plot>
+  <plot>Rocketman blasts out of Water Seven and connects with the sea tracks, and old foes become allies for the coming struggle. Aqua Laguna swells, and it's up to Luffy and Zoro to combine their might and create safe passage for all aboard.
+
+Manga Chapter(s): 366-367
+
+Anime Episode(s): 256-257</plot>
   <runtime>0</runtime>
   <mpaa/>
-  <premiered/>
-  <aired/>
+  <premiered>2024-02-16</premiered>
+  <aired>2024-02-16</aired>
```

Comment:
This should be `2024-02-17` as displayed on [onepace.net/watch](https://www.onepace.net/watch) in the episode description